### ISSUE TITLE
Check Oreo notification updates

### ISSUE THREAD

**Issue by ognjenns** [RestComm/restcomm-android-sdk#751]

**Comment by ognjenns** [comment]
I have implemented regular notifications for Oreo devices. But I read about the https://developer.android.com/reference/android/telecom/ConnectionService.html and Its similar thing like CallKit on iOS. I was reading documentation and there is no much details. I think we should implement this in the future. 
Anyhow, the pull request for regular push with Oreo:
https://github.com/RestComm/restcomm-android-sdk/pull/792

**Comment by atsakiridis** [comment]
- Great @ognjenns, so with this fix are we covered 100% for Oreo? Or are there any edge cases that we are having troubles with? For example are we good with the scenario we discussed 1-2 weeks back where the call stays ringing in the background until the user responds using notification drawer? Does Oreo allow for the SDK to be kept alive in the background until the user answers or does it kill it?
- Sure we can discuss more about ConnectionService after vacation to see if it makes sense
- Will check PR when I get back from vacation, meanwhile, keep 'em coming ;)
